recipes/doxygen/all/conanfile.py

```diff
@@ -77,6 +77,10 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
 
+    def package_id(self):
+        self.info.include_build_settings()
+        del self.info.settings.compiler
```

Comment:
```suggestion
        del self.info.settings.build_type
        del self.info.settings.compiler
```

- We don't need to build a Debug doxygen executable.
- I don't think there is a need to do `self.info.include_build_settings()`. `arch_build` and `os_build` are not included in `settings`.
See https://github.com/conan-io/conan/blob/f230bf104842c160be6ea377497151fe1ea64c38/conans/model/info.py#L628-L630